cluster-autoscaler/main.go

```diff
@@ -171,6 +171,7 @@ var (
 			"Pods with nominatedNodeName set are always filtered out.")
 
 	ignoreTaintsFlag         = multiStringFlag("ignore-taint", "Specifies a taint to ignore in node templates when considering to scale a node group")
+	ignoreLabelsFlag         = multiStringFlag("ignore-labels", "Specifies a label to ignore when comparing if two node groups are similar")
```

Comment:
State in help that this this list extends a set of basic and cloud provider specific labels which are always ignored.